<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WLMPGWidget</class>
 <widget class="QWidget" name="WLMPGWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>124</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777214</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBoxX">
     <property name="toolTip">
      <string>key Q,W,E</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbX1">
        <property name="text">
         <string>x1</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupX</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rbX10">
        <property name="text">
         <string>x10</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupX</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="rbX100">
        <property name="text">
         <string>x100</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupX</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxV">
     <property name="toolTip">
      <string>key V</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="cbVmode">
        <property name="text">
         <string>V mode</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="groupBoxAxis">
     <property name="toolTip">
      <string>key 0,1...6</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="5">
       <widget class="QRadioButton" name="rb5">
        <property name="text">
         <string>5</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupAxis</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rbX">
        <property name="text">
         <string>X</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupAxis</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="rbZ">
        <property name="text">
         <string>Z</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupAxis</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbOff">
        <property name="text">
         <string>OFF</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupAxis</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QRadioButton" name="rb4">
        <property name="text">
         <string>4</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupAxis</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="rbY">
        <property name="text">
         <string>Y</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupAxis</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QRadioButton" name="rb6">
        <property name="text">
         <string>6</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupAxis</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupAxis"/>
  <buttongroup name="buttonGroupX"/>
 </buttongroups>
</ui>
